frontend/src/authentication/actions/index.js

```diff
@@ -0,0 +1,6 @@
+export const AuthActions = {
+    AUTH_USER: 'AUTH_USER',
+    UNAUTH_USER: 'UNAUTH_USER',
+    AUTH_ERROR: 'AUTH_ERROR',
```

Comment:
Rename actions. SIGN_IN_* wold be better names. (see what prefixes are used for async actions) I'd also suggest to rename login to SignIn everywhere.